src/contexts/tour-context/context.tsx

```diff
@@ -0,0 +1,876 @@
+            floatingUIOptions: {
+                middleware: [
+                    tooltipOffset({ mainAxis: 16 })
+                ]
+            },
+            floatingUIAutoUpdateOptions: {
+                animationFrame: true
+            },
+            buttons: [
+                {
+                    classes: secondaryButtonClass,
+                    text: 'Back',
+                    type: 'back'
+                },
+                {
+                    classes: primaryButtonClass,
+                    text: 'Next',
+                    type: 'next'
+                }
+            ]
+        },
+        useModalOverlay: true
+    }), [])
+    
+    // NOTE: we can't really access internal React context here because of how shepherd works.
+    const tourSteps = useMemo<ShepherdOptionsWithTypeFixed[]>(() => {
+        let error: string | null = null
+        // Message can be HTML markup
+        const doError = (message?: string) => {
+            if (error) return
+            if (!message) message = renderToStaticMarkup(
+                <div>
+                    Something unexpected went wrong during the tour.
+                </div>
+            )
+            error = message
+            // Cull every step after the current step.
+            tour.steps.splice(tour.steps.indexOf(tour.getCurrentStep()!) + 1, tour.steps.length)
+            // Add an error step
+            tour.addStep({
+                id: "error",
+                attachTo: {
+                    element: "head"
+                },
+                showOn: () => error !== null,
+                title: () => "Sorry!",
+                text: () => error!,
+                buttons: [
+                    {
+                        classes: secondaryButtonClass,
+                        text: 'Exit',
+                        action: () => tour.cancel()
+                    },
+                    {
+                        classes: primaryButtonClass,
+                        text: 'Reload',
+                        action: () => window.location.reload()
+                    }
+                ]
+            })
+            // Advance to error step.
+            tour.next()
+        }
+        if (context.brand === "heal") return [
+            {
+                id: "main.intro",
+                path: homePathPattern,
+                attachTo: {
+                    // We want to mount to a sizeless element so that nothing is "highlighted",
+                    // i.e., everything is backdrop except the floating tour popup.
+                    element: "head"
+                },
+                title: "Welcome to HEAL Semantic Search (HSS)",
+                text: renderToStaticMarkup(
+                    <div>
+                        HSS is an open-access search engine for exploring the HEAL research landscape.<br /><br />
+                        HSS is not the search you use every day - it uses linked knowledge to return concepts related
```

Comment:
The "HSS is not the search you use every day" line annoys me, because Google Search (and, I'm guessing, Bing and DuckDuckGo and all the others) have a lot of semantic search features these days! But I guess this is the approved text from the HEAL Stewards.